.github/workflows/containers.yml

```diff
@@ -79,3 +79,37 @@ jobs:
             ghcr.io/spack/${{ matrix.dockerfile[0] }}:latest
             ${{ steps.meta.outputs.tags }}
           cache-to: type=inline
+
+  upload-gcc-buildcache:
+    timeout-minutes: 60
+    runs-on: self-hosted
+    permissions:
+      packages: write
+    needs: [ deploy-images ]
+    env:
+      CONTAINER_NAME: 'ghcr.io/spack/pcluster-amazonlinux-2'
+    steps:
+      - uses: actions/checkout@v3
+      - run: |
+          id=$(docker create --platform linux/amd64 ${CONTAINER_NAME})
```

Comment:
I think here we must recompute (or pass from previous job) the tag of the PR image. Otherwise we'll use `latest`, right?

Reply:
Yes, I need to find out how to do that.

Reply:
I think I got it now.